src/webauthn/src/Event/AuthenticatorAssertionResponseValidationFailedEvent.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Webauthn\Event;
+
+use Psr\Http\Message\ServerRequestInterface;
+use Throwable;
+use Webauthn\AuthenticatorAssertionResponse;
+use Webauthn\PublicKeyCredentialRequestOptions;
+
+class AuthenticatorAssertionResponseValidationFailedEvent
+{
+    public function __construct(
+        private readonly string $credentialId,
```

Comment:
Is it a good idea to keep getters when we can directly have `public readonly` properties?
Should we drop these boring methods for all events?

Reply:
I let them as there are. Maybe we can clean all events in a new MR ?

Reply:
ok👌